modules/network-annotation-processor/src/main/java/org/apache/ignite/internal/network/processor/messages/MarshallableTypesBlackList.java

```diff
@@ -97,11 +126,17 @@ public Boolean visitDeclared(DeclaredType t, Void unused) {
                 return t.getTypeArguments().stream().anyMatch(this::visit);
             }
 
-            return !isSameType(NATIVE_TYPES, t) && !typeUtils.isSubType(t, NetworkMessage.class);
+            return !isSameType(NATIVE_TYPES, t)
+                    && !typeUtils.isSubType(t, NetworkMessage.class)
```

Comment:
Would it make sense to implement NetworkMassage and default types as a blacklist file themselves?